protos/feast/core/JobService.proto

```diff
@@ -74,15 +74,19 @@ message Job {
   JobStatus status = 3;
   // Deterministic hash of the Job
   string hash = 8;
+  // Start time of the Job
+  google.protobuf.Timestamp start_time = 9;
 
   message RetrievalJobMeta {
     string output_location = 4;
   }
 
   message OfflineToOnlineMeta {
+      string feature_table = 10;
```

Comment:
Since it's nested message not sure indexing should be dependent on parent?
`string feature_table = 1;` should work just fine